swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from .item import Item
+
+# wave 1
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = [] if inventory is None else inventory
```

Comment:
Would be slightly more Pythonic to check `if not inventory` 
```suggestion
        self.inventory = [] if not inventory else inventory
```